sources/web/datalab/login.ts

```diff
@@ -1,109 +0,0 @@
-/*
```

Comment:
You're absolutely right.

I've sent out https://github.com/googledatalab/datalab/pull/1414 to remove it separately.